# 2003-04-07  Roman Zippel  <zippel@linux-m68k.org>
# 
# 	PR optimization/9812
# 	* config/m68k/m68k.md: Avoid forcing fp constants into memory.

--- gcc/config/m68k/m68k.md	17 Mar 2003 10:22:00 -0000	1.48.8.1.4.3
+++ gcc/config/m68k/m68k.md	7 Apr 2003 18:05:03 -0000
@@ -675,7 +675,7 @@
 
 (define_insn ""
   [(set (cc0)
-	(compare (match_operand:DF 0 "general_operand" "f,mG")
+	(compare (match_operand:DF 0 "general_operand" "f,fmG")
 		 (match_operand:DF 1 "general_operand" "fmG,f")))]
   "TARGET_68881"
   "*
